ckanext/ontario_theme/fanstatic/common.css

```diff
@@ -3492,4 +3492,8 @@ label::after {
     align-items: center;
     gap: 20px;
   }
+}
+
+.form-actions {
+  overflow: unset;
 }
```

Comment:
You can add `text-align: unset;` here and achieve the same result as adding the class `.pull-left` and remove the class from the button. What do you think?